WORKSPACE

```diff
@@ -119,8 +119,8 @@ http_archive(
     build_file_content = "# empty",
     patch_args = ["-p1"],
     patches = [
-        "@carbon//bazel/llvm_patches:0001_Patch_for_mallinfo2_when_using_Bazel_build_system.patch",
-        "@carbon//bazel/llvm_patches:0002_Added_Bazel_build_for_compiler_rt_fuzzer.patch",
+        "@carbon//bazel/pkg_patches/llvm:0001_Patch_for_mallinfo2_when_using_Bazel_build_system.patch",
```

Comment:
Looking at the precommit error [here](https://github.com/carbon-language/carbon-lang/actions/runs/3687766633/jobs/6241754700), I think /.pre-commit-config.yaml needs to be edited. I've sent out #2469 for this and will try to get it in quickly for you.

As part of that, can we also call the dir just "patches" instead of "pkg_patches"? Bazel's package concept doesn't quite match what I think you intend here; I think just "patches" is good with the subdirectories as you suggest.

Reply:
#2469 is now in. Can you merge the changes in here, so that the pre-commit is passing?

Reply:
done